### PR TITLE
Stop watching the invoice when the modal is closed

### PR DESCRIPTION
**Pull request by bumi**
updated version of #55 
seems the widget is initialized on every screen change thus we use a global variable to store the interval id.

closes #55

### CODE REVIEW

File: src/services/invoice.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 export default class InvoiceService {
   constructor(options) {
-    this.baseURL = options.baseURL;
-    this.memo = options.memo || "";
+    console.log(options);
```
Comment (reznord) [inline src/services/invoice.js:3]: nit pick: `console.log`

File: src/services/invoice.js [hunk 2]
```diff
@@ -100,6 +107,7 @@ export default class InvoiceService {
       if (response.ok) {
         return response.json();
       } else {
+        console.log(response);
```
Comment (reznord) [inline src/services/invoice.js:110]: nit pick: `console.log`

File: src/services/invoice.js (hunk 1 above)
Comment (bumi) [reply src/services/invoice.js:3]: uuups. thanks!

File: src/containers/widget/index.js [hunk 3]
```diff
@@ -26,6 +26,11 @@ class Widget extends Component {
       isModalOpen: props.showModal,
       currentScreen: props.screenName,
     };
+    this.invoiceService = new InvoiceService({
```
Comment (reznord) [inline src/containers/widget/index.js:29]: moving to class made this much easier 😄

**Review by reznord — APPROVED** [review]

**Pull request merged**